Hearthstone Deck Tracker/GameEventHandler.cs

```diff
@@ -1072,6 +1072,7 @@ private async void HandleConstructedStart()
 
 		private async void HandleBattlegroundsStart()
 		{
+			Core.Overlay.ShowBgsTopBar();
```

Comment:
We should wait for the game to be faded in before we show the UI. Below is a 3 second delay; this should probably go into the else case after that.